components/organisms/TopicViewerContent.tsx

```diff
@@ -112,7 +112,7 @@ export default function TopicViewerContent({ topic, onEnded, offset }: Props) {
           ...authors(topic),
         ]}
       />
-      <Box component="article" sx={{ my: -1.5 }}>
+      <Box component="article">
         <Markdown>{topic.description}</Markdown>
       </Box>
```

Comment:
スタイルシートが不要ならばもはやBox コンポーネントはおそらく不要で、素朴なHTMLセマンティクスのみでも良さそうですね。

```suggestion
      <article>
        <Markdown>{topic.description}</Markdown>
      </article>
```